React/Views/UIView+React.h

```diff
@@ -113,7 +113,7 @@
  * transparent in favour of some subview.
  * Defaults to `self`.
  */
-@property (nonatomic, readonly) RCTPlatformView *reactAccessibilityElement; // TODO(macOS ISS#2323203)
+@property (nonatomic, readonly) id *reactAccessibilityElement; // TODO(macOS ISS#2323203)
```

Comment:
`id` is a pointer, so it should be `id reactAccessiblityElement` instead of `id *reactAccessibilityElement`

Reply:
My bad.